admin-client/src/main/java/com/epam/admin_client/config/CustomAuthenticationProvider.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.admin_client.config;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.interfaces.Claim;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.epam.admin_client.service.LoginService;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CustomAuthenticationProvider implements AuthenticationProvider {
+
+    private static final String ADMIN = "ADMIN";
+
+    @Autowired
+    private LoginService loginService;
+
+    @Override
+    public Authentication authenticate(Authentication authentication)
+        throws AuthenticationException {
+
+        String email = authentication.getName();
+        String password = authentication.getCredentials().toString();
+        String token = loginService.authenticate(email, password);
+
+        DecodedJWT decode = JWT.decode(token.substring(7));
```

Comment:
LoginService returns token with attached "Bearer " to it. In this component you immediately throw Bearer away. What was the point to attach Bearer at all?